src/helpers/api.ts

```diff
@@ -1,28 +1,45 @@
-import {
-  FPJS_SECRET_TOKEN,
-  FPJS_VISITORS_ENDPOINT,
-  FPJS_MGMT_API_HOST,
-  BOTD_TOKEN_ENDPOINT,
-  GREENHOUSE_COMPANY_ID,
-} from '../constants/env'
+import { FPJS_SECRET_TOKEN, FPJS_VISITORS_ENDPOINT, FPJS_MGMT_API_HOST, GREENHOUSE_COMPANY_ID } from '../constants/env'
 
 export async function loadFpjsHistory(visitorId: string) {
   // 21 to show >20 on homepage
   const response = await fetch(`${FPJS_VISITORS_ENDPOINT}${visitorId}?token=${FPJS_SECRET_TOKEN}&limit=21`)
   return await response.json()
 }
 
-export async function createNewLead(
-  formName: string,
-  email: string,
-  url: string,
-  jobTitle: string,
-  description: string,
-  landingPage: string,
-  previousPage: string,
-  utmParams: Record<string, string>,
+export enum ipRegion {
```

Comment:
Enums should use capital case
```suggestion
export enum IpRegion {
```